components/devtools/actors/inspector/node.rs

```diff
@@ -166,13 +170,46 @@ impl NodeInfoToProtocol for NodeInfo {
             actors.script_to_actor(self.unique_id)
         };
 
+        let name = actors.actor_to_script(actor.clone());
+
+        // If a node only has a single text node as a child whith a small enough text,
+        // return it with this node as an `inlineTextChild`.
+        let inline_text_child = (|| {
+            // TODO: Also return if this node is a flex element
+            if self.num_children != 1 || self.node_name == "SLOT" {
+                return None;
+            }
+
+            let (tx, rx) = ipc::channel().ok()?;
+            script_chan
+                .send(GetChildren(pipeline, name.clone(), tx))
+                .unwrap();
+            let mut children = rx.recv().ok()??;
+
+            let child = children.pop()?;
+            let msg = child.encode(actors, true, script_chan.clone(), pipeline);
+
+            // Check if it is a text node
+            if msg.node_type != 3 {
```

Comment:
It would be better to have a constant for this magic number. A comment would be useful as well explaining that the constant is defined again to avoid a dependency on `script` where it is actually defined (`script::dom::bindings::codegen::Bindings::NodeBinding::NodeConstants`).